### ISSUE TITLE
Replace old Social-Banner Images

### ISSUE THREAD

**Issue by thecodemonkey** [lokalkauf/lokalkauf-frontend#69]
the Social Banner(Meta tags og:image) contains the old images. These must be replaced by the new logos.

**Comment by MaximilianSei** [comment]
[](url)https://allfacebook.de/pages/bildgroessen-facebook-twitter-google-pinterest/

**Comment by christianaschoff** [comment]
nice page from a design perspective. Out main problem is that we only provide one site with nearly 20-30 meta tags just for that social stuff. And f.e. twitter does not allow multiple formats on a single page. You must decide what format you want to support.

**Comment by MaximilianSei** [comment]
I need to know the formats for the different channels. I don´t whta to look for in the web. Please help me!

**Comment by christianaschoff** [comment]
yeah, that is the point. We can provide any size of image, but it needs to fit for all media outlets. That the big challenge to find the right aspect ratio and the correct content placement for our usecase. IMHO are people with design and social media background more qualified to answer that question.

TL;DR we can provide any size, but there are certain recommendation I guess. But we need to dig deeper into that subject.

**Comment by MaximilianSei** [comment]
Größenanforderungen für Feature-Bilder

Wenn das Feature-Bild für einen Link, den Sie eingefügt haben, nicht korrekt in Ihrer Social-Media-Vorschau angezeigt wird (z. B. wird das Bild abgeschnitten oder das Verhältnis ist nicht korrekt), müssen Sie das Bild möglicherweise so bearbeiten, dass es die richtige Größe für das soziale Netzwerk hat, in dem Sie einen Beitrag posten möchten, und dann die Seite erneut veröffentlichen. Zur Optimierung des Feature-Bildes auf den einzelnen Netzwerken werden die folgenden Maße empfohlen:

Facebook: 300:175
Twitter: 16:9 für Fotobeiträge und 1,91:1 für Feature-Bilder in Link-Beiträgen
Instagram (Quadrat): 1:1
Instagram (Hochformat): 4:5
Instagram (Querformat): 1.91:1
LinkedIn (Querformat): 1.91:1

**Comment by MaximilianSei** [comment]
Do we really need one fits all or do do we need to provide a picture for every channel? Why we have a new picture in Twitter, but old pictures in FB, WA and Insta?
https://de.ryte.com/product-insights/die-perfekte-social-vorschau-mit-den-ryte-social-reports

**Comment by christianaschoff** [comment]
twitter and "other" point to the same image provided by us. But, some SOM cache images and you need to bust them within the platform. But I guess Twitter is known to have that issue. That is the image that is used right now 
![lokalkaufSocialMedia](https://user-images.githubusercontent.com/13931688/81106184-6f24f680-8f15-11ea-8869-a0b57babac91.png)

**Comment by MaximilianSei** [comment]
![LK_TeaserBilder_200512_eco3](https://user-images.githubusercontent.com/64410346/81940681-239ad880-95f8-11ea-8e5b-43d1a3f5b60f.jpg)
or
https://drive.google.com/drive/u/1/folders/1jj1OrnJnsG_UkOmslTsiHDFgtyhPrTsw

**Comment by MaximilianSei** [comment]
Please try this one! It might be that we need to put the logo in the middle and have enough space on the right and left, but lets try as it has the the right ratio format. Can we test it with staging?

**Comment by MaximilianSei** [comment]
Try just the logo
https://drive.google.com/file/d/1ppLt3AJc34IugoZ9BB_KiTxkKbSoSJ15/view

**Comment by christianaschoff** [comment]
is that image already exchanged?

**Comment by owlRike** [comment]
On Facebook and WhatsApp it's still the old logo.
Anything else you need to be able to fix it?

**Comment by christianaschoff** [comment]
that is because you need to provide an absolute link to the resource.
So it is pointing to the social image on production even on staging :)

**Comment by owlRike** [comment]
So it cannot be tested?

**Comment by christianaschoff** [comment]
I fear so

**Comment by thecodemonkey** [comment]
You cannot test it directly, but after the release it should work. The stored pictures look good.
